src/adminEntity.ts

```diff
@@ -5,11 +5,13 @@ import DefaultAdminSite from './adminSite'
 import ManyToManyWidget from './widgets/manyToManyWidget'
 import InvalidDisplayFieldsException from './exceptions/invalidDisplayFields.exception'
 import { WidgetConstructor } from './widgets/widget.interface'
+import { Request, Response } from 'express'
 
+type listActions = { label: string; action: (request: Request, response: Response) => void }
```

Comment:
This represents a single action, so it should be singular. Also, types are capitalised by convention:
```suggestion
type ListAction = { label: string; action: (request: Request, response: Response) => void }
```

You also need to export that, and also export it from the root `index.ts` so that users can do `import { ListAction } from 'nestjs-admin'`